app/src/signing/networkMessages/polkadot.js

```diff
@@ -125,30 +125,24 @@ export async function RestakeTx(
   return await getSignMessage(senderAddress, transactions, api)
 }
 
-export async function ClaimRewardsTx(senderAddress, {}, network, api) {
+export async function ClaimRewardsTx(senderAddress, { rewards }, network, api) {
```

Comment:
I explicitly not did this in here to release the API first as the PR name suggests

Reply:
:sweat_smile:

Reply:
oh I see, didn't realize this API changes break the FE

Reply:
please revert the FE changes

Reply:
give me a sec

Reply:
Done